patches/chrome-browser-importer-external_process_importer_client.cc.patch

```diff
@@ -1,17 +1,12 @@
 diff --git a/chrome/browser/importer/external_process_importer_client.cc b/chrome/browser/importer/external_process_importer_client.cc
-index 5c56fb75075438213e2ec261ffe4c1833a5c8d16..bbc7ce6ef5cee3b48f0910ea8f278998507a55ff 100644
+index 5c56fb75075438213e2ec261ffe4c1833a5c8d16..1e4d774a3058688e3bda6ca6b61296c0cc816383 100644
 --- a/chrome/browser/importer/external_process_importer_client.cc
 +++ b/chrome/browser/importer/external_process_importer_client.cc
-@@ -68,6 +68,12 @@ void ExternalProcessImporterClient::Start() {
+@@ -68,6 +68,7 @@ void ExternalProcessImporterClient::Start() {
    localized_strings.try_emplace(
        IDS_BOOKMARK_BAR_FOLDER_NAME,
        l10n_util::GetStringUTF8(IDS_BOOKMARK_BAR_FOLDER_NAME));
-+  localized_strings.try_emplace(
-+      IDS_IMPORT_FROM_BRAVE,
-+      l10n_util::GetStringUTF8(IDS_IMPORT_FROM_BRAVE));
-+  localized_strings.try_emplace(
-+      IDS_BOOKMARK_GROUP_FROM_BRAVE,
-+      l10n_util::GetStringUTF8(IDS_BOOKMARK_GROUP_FROM_BRAVE));
++  BRAVE_START
```

Comment:
😎👌

Reply:
can't we do this in `StartImport` instead of patching?

Reply:
Yes, I think it's more complicated and maybe it also needs another patching.
To do that, we have to subclass ProfileImportProxy::StartImport() and it should be replaced with BraveProfileImportProxy in `chrome/common/importer/profile_import.mojom.h`.
So, I think one line patching here is better. WDYT?

Reply:
Why would you need to patch the mojo stuff? Just subclass ProfileImportImpl and chromium_src override in chrome/utility/services.cc

Reply:
Ah, ok. I didn't think to override in utility process side.
and I found `ProfileImportImpl` also has some patches that could be cleanup together :)